flake.nix

```diff
@@ -56,15 +55,16 @@
 
         pkgs = import nixpkgs {
           inherit system;
-          overlays = [ rust-overlay.overlays.default wasm-bindgen-cli-overlay ];
+          overlays = [ inputs.rust-overlay.overlays.default wasm-bindgen-cli-overlay ];
         };
 
-        craneLib = crane.mkLib pkgs;
+        craneLib = inputs.crane.mkLib pkgs;
 
-        tree-sitter-nickel = tree-sitter-nickel-input.packages.${system}.default;
+        tree-sitter-nickel = inputs.tree-sitter-nickel.packages.${system}.default;
 
         topiaryPkgs = pkgs.callPackage ./default.nix {
-          inherit advisory-db crane rust-overlay nix-filter craneLib tree-sitter-nickel;
+          inherit (inputs) advisory-db crane rust-overlay nix-filter tree-sitter-nickel;
+          inherit craneLib;
```

Comment:
This changes the semantics of the previous code. You probably want to move `tree-sitter-nickel` on the second `inherit` line.

Reply:
I KNEW IT ;-)